Mathlib/Algebra/Algebra/Defs.lean

```diff
@@ -175,11 +175,13 @@ def RingHom.toAlgebra' {R S} [CommSemiring R] [Semiring S] (i : R →+* S)
   smul_def' _ _ := rfl
   toRingHom := i
 
+/-- definition of smul of toAlgebra'-/
```

Comment:
here I think you can just use `set_option linter.docPrime false in`, since your excuse is that you're writing a lemma about a declaration that is itself primed.